packages/templates/src/raw/celebrity-q-and-a/index.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { __, _x } from '@web-stories-wp/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { default as template } from './template';
+
+export default {
+  title: _x('Celebrity Q & A', 'template name', 'web-stories'),
+  tags: [
+    _x('Entertainment', 'template keyword', 'web-stories'),
+    _x('Story', 'template keyword', 'web-stories'),
```

Comment:
This keyword seems redundant... Obviously it's a (web) story.

Reply:
Could it be that the word `Story` here may not be indicative of a Web Story but actual story, like story, tale telling. Not sure though 😅 . Can be removed as seems ambiguous.

Reply:
Hmm yeah problably. Perhaps replace with `Interview`?